src/test/java/org/jenkinsci/plugins/codesonar/integration/conditions/WarningCountIncreaseSpecifiedScoreAndHigherConditionIT.java

```diff
@@ -58,6 +61,8 @@ public void percentageOfWarningsBelowTheDesignatedRankIsBellowTheThreshold_Build
         // arrange
         final Result EXPECTED_RESULT = Result.SUCCESS;
         final String WARRANTED_RESULT = Result.FAILURE.toString();
+        final String VISIBILITY_FILTER = "active"; // active warnings
+        final String VISIBILITY_FILTER_NEW_WARNINGS = "all"; // all warnings
```

Comment:
(same comments as above)

1. The comments "active warnings", "all warnings" are unnecessary and should be removed.
2. Should `VISIBILITY_FILTER_NEW_WARNINGS` be set to to "new" and not "all"?